en/features_mc/throw_launch.md

```diff
@@ -0,0 +1,92 @@
+# Throw Launch (Multicopter)
+
+:::warning Experimental
+This feature was introduced after PX4 v1.14.
+
+- It has not yet been broadly tested on different vehicle configurations or scenarios. various platforms and in different scenarios.
+- The majority of testing has been done in position mode.
+  Other modes should also work.
+
+:::
+
+This feature allows a multicopter to be started by arming it from a fixed position and then throwing it into the air.
+The vehicle then turns on the motors and operates according to its current mode.
+
+The vehicle detects that it has been thrown based on reaching a certain speed (5m/s), and then starts the motors at the apex of the throw, once it determines that it has started to fall.
+You need to throw it high enough so that the vehicle can stabilize its height well before falling in range of people (2 or 3 metres is usually sufficient).
```

Comment:
This might suggest to some that it's okay to throw the drone 2m away from people, I try to keep at least 10-20m even when flying normally... Maybe we can just remove "in range of people"?

Reply:
I agree maybe "in range of any obstacles" is less tempting to bring people close.

Reply:
Good catch. It really depends on your vehicle - if you had a small drone with prop guards it wouldn't matter. Let's do this.

```suggestion
You need to throw it high enough so that the vehicle can stabilize its height well before falling anywhere near people.
```